src/shared/seccomp-util.c

```diff
@@ -1192,6 +1192,27 @@ int seccomp_restrict_realtime(void) {
         return 0;
 }
 
+static int add_seccomp_syscall_filter(scmp_filter_ctx seccomp,
```

Comment:
still think we should mark functions specially if they log but are "library calls", like this one is...

And the fact that this one only does EPERM rules sounds like something to mention too...

but i figure this doesn't matter much